charts/k3s/templates/pdb.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if and (.Values.enableHA) (gt (int .Values.replicas) 1) -}}
+{{- if .Capabilities.APIVersions.Has "policy/v1" -}}
+apiVersion: policy/v1
+{{- else }}
+apiVersion: policy/v1beta1
```

Comment:
I don't have a strong opinion on this. Since the PodDisruptionBudget in the policy/v1beta1 API version is [removed in Kubernetes v1.25](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125) and is supported since v1.21, we can also remove the conditional here.

Reply:
Yes, please remove that check

Reply:
Done in [ce3d8e5](https://github.com/loft-sh/vcluster/pull/795/commits/ce3d8e53c22076d0560607a80529e5e974c0857e)